ts/modules/wordcloud.src.ts

```diff
@@ -980,14 +980,21 @@ var wordCloudSeries: Partial<Highcharts.WordcloudSeries> = {
                 return p.weight;
             }),
             maxWeight = Math.max.apply(null, weights),
-            data = series.points.sort(function (
+            // concat() prevents from sorting the original array.
+            data = series.points.concat().sort(function (
                 a: Highcharts.WordcloudPoint,
                 b: Highcharts.WordcloudPoint
             ): number {
                 return b.weight - a.weight; // Sort descending
             }),
             field: Highcharts.WordcloudFieldObject;
 
+        // Reset the scale before finding the dimensions (#11993)
```

Comment:
A comment why we need to reset the scale would be nice. It's quite unusual reason AFAIR ;)